libs/runtime_local/src/runtime_local.cpp

```diff
@@ -952,9 +952,8 @@ namespace hpx {
             return false;
         }
 
-        bool numa_sensitive = false;
         if (std::size_t(-1) != get_worker_thread_num())
-            return numa_sensitive;
+            return false;
```

Comment:
Hmm, does this function ever return anything but `false`? Either this function should query the default/current scheduler for the value or the function shouldn't exist at all. Does not need to be changed in this PR.

Reply:
I didn't look at the semantics here. Just removed the variable that was not needed anymore.

Reply:
No worries, let's not mix it into this PR.